data/translations/fr/homepage.json

```diff
@@ -0,0 +1,28 @@
+{
+  "banner__banner_text": "Un site Web officiel du gouvernement américain",
+  "banner__banner_action": "Voici comment vous le savez",
+  "banner__guidance_gov_heading": "Les sites Web officiels utilisent .gov",
+  "banner__guidance_gov_text": "Un site Web %.gov% appartient à une organisation gouvernementale officielle aux États-Unis.",
+  "banner__guidance_https_heading": "Les sites Web .gov sécurisés utilisent HTTPS",
+  "banner__guidance_https_text": "Un verrou %lock_icon% ou %https% signifie que vous êtes connecté(e) en toute sécurité au site Web .gov. Assurez-vous de ne partager des informations sensibles que sur des sites Web officiels et sécurisés.",
+  "alert__alert_msg": "En raison du COVID-19, il peut y avoir des changements concernant quand, où et comment vous votez aux prochaines élections. Consultez le site Web électoral de votre État pour en savoir plus.",
+  "alert__alert_after_election": "Pour les résultats des élections, visitez le site Web électoral de votre État.",
+  "header": "Inscrivez-vous pour voter",
+  "state_selection__heading": "Où vivez-vous ?",
+  "state_selection__default": "Sélectionnez votre État ou territoire",
+  "state_selection__submit": "Découvrez comment vous inscrire",
+  "registered__header": "Déjà inscrit(e) ?",
+  "registered__resources_intro": "Découvrez nos %link% au %USA.gov%",
+  "registered__resources_link_text": "ressources sur le vote et les élections",
+  "registered__resources_link_url": "https://www.usa.gov/voting",
+  "registered_links__electionday": "Voter le jour des élections",
+  "registered_links__electionday_url": "https://www.usa.gov/election-day",
+  "registered_links__voterid": "Exigences d'identification pour voter",
+  "registered_links__voterid_url": "https://www.usa.gov/voter-id",
+  "registered_links__process": "Processus électoral",
+  "registered_links__process_url": "https://www.usa.gov/election",
+  "registered_links__absentee": "Vote par correspondance",
```

Comment:
Add (in english) translation to the end of this string for the untranslated site link.